tasks/populate-release-notes-images/populate-release-notes-images.yaml

```diff
@@ -78,13 +78,15 @@ spec:
                 # purl should be pkg:oci/bar@sha256%3Aabcde?arch=amd64&repository_url=registry.redhat.io/foo
                 purl="pkg:oci/${deliveryRepo##*/}@${digest/:/%3A}?arch=${arch}&repository_url=${deliveryRepo%/*}"
                 jsonString=$(jq -cn \
+                    --arg component "$name" \
```

Comment:
Please bump the task version and add a changelog entry to README for this.

Reply:
You should also add a check for this to at least one test - maybe `test-populate-release-notes-images-single-image`.

Reply:
test added

Reply:
w.r.t. the task version, I'm waiting for #548 to be merged to avoid conflicts.

Reply:
548 merged